### PR TITLE
Allow working with the wallet when offline

### PR DESCRIPTION
**Pull request by nop33**
# sound: ON! 🔉 

https://user-images.githubusercontent.com/1579899/157693104-41009aed-0de2-4043-9daf-f5e9400e89c8.mp4

https://user-images.githubusercontent.com/1579899/157693129-5cab1cef-ea5b-478a-99fb-9d63b73b4edf.mp4

### CODE REVIEW

File: src/pages/Wallet/WalletLayout.tsx [hunk 1]
```diff
@@ -85,14 +86,24 @@ const WalletLayout: FC = ({ children }) => {
...
   if (!wallet) return null
 
   return (
     <WalletContainer initial={{ opacity: 0 }} animate={{ opacity: 1 }} transition={{ duration: 0.8 }}>
       <AppHeader>
-        <RefreshButton transparent squared onClick={refreshData} disabled={isLoadingData} aria-label="Refresh">
-          {isLoadingData ? <Spinner /> : <RefreshCw />}
-        </RefreshButton>
+        {isOffline ? (
+          <OfflineIcon data-tip={'The wallet is offline. Trying to reconnect...'}>
```
Comment (mvaivre) [inline src/pages/Wallet/WalletLayout.tsx:99]: We can use double quotes.

**Comment by mvaivre** [comment]
For info: I'll use this PR to add tooltips with labels for each icon in the `AppHeader`.

**Comment by nop33** [comment]
Now that you solved your own feedback (😅) can we merge this one?

**Comment by mvaivre** [comment]
Yep!

**Pull request merged**